ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ingress/custom-ingresscontroller/provision/cucushift-installer-rehearse-azure-ipi-ingress-custom-ingresscontroller-provision-chain.yaml

```diff
@@ -0,0 +1,18 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-ingress-custom-ingresscontroller-provision
+  steps:
+  - ref: azure-provision-resourcegroup
+  - ref: azure-provision-vnet
+  - ref: ipi-conf
+  - ref: ipi-conf-telemetry
+  - ref: ipi-conf-azure
+  - ref: ipi-conf-azure-provisionednetwork
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-install-rbac
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-install
+  - ref: ipi-install-times-collection
+  - ref: enable-qe-catalogsource
+  - ref: ingress-azure-custom-ingresscontroller
```

Comment:
Sound like this is kind of post-actions for installation, if that, we can replace the above lines with `chain: cucushift-installer-rehearse-azure-ipi-provision`

Reply:
Yes, it is post-action for installation but it needs to create new subnet so the ref `azure-provision-resourcegroup, azure-provision-vnet, ipi-conf-azure-provisionednetwork` are required, if using `chain: cucushift-installer-rehearse-azure-ipi-provision` then we cannot create new subnet in the default vnet.

Reply:
Good to know.